rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -934,6 +949,7 @@ class ProcMatcherSpec extends FlatSpec with Matchers {
 
     val proc = new PInput(receipt, new PNil())
 
+    // format: off
```

Comment:
This is for `scalamft` -- it has problems when there are too many nested parentheses and reports `state exploded` error. The solution proposed by creators of it is to turn off formatting around offending lines.